config/locales/hyrax.it.yml

```diff
@@ -1063,12 +1063,12 @@ it:
   simple_form:
     hints:
       admin_set:
-        description: Una breve descrizione generale che si applica a tutte le opere raccolte in questo set. Ad esempio, "Tesi e file supplementari creati dagli studenti laureati della School of Earth Sciences".
+        description: Una breve descrizione generale che si applica a tutte le opere raccolte in questo set. Ad esempio, 'Tesi e file supplementari creati dagli studenti laureati della School of Earth Sciences'.
         title: Un nome per facilitare l'identificazione del set amministrativo e per distinguerlo dagli altri set amministrativi nel repository.
       collection:
         based_near: Un nome di luogo correlato alla raccolta, ad esempio il sito di pubblicazione o la città, lo stato o il paese di cui sono contenuti i contenuti della raccolta. Invita il <a href='http://www.geonames.org'> servizio web GeoNames </a>.
         contributor: Una persona o un gruppo che desideri riconoscere per avere un ruolo nella creazione della collezione, ma non il ruolo principale.
-        creator: La persona o il gruppo responsabile della raccolta. Di solito questo è l'autore del contenuto. I nomi personali devono essere inseriti prima con il cognome, ad es. "Smith, John."
+        creator: La persona o il gruppo responsabile della raccolta. Di solito questo è l'autore del contenuto. I nomi personali devono essere inseriti prima con il cognome, ad es. 'Smith, John.'
```

Comment:
because there are single quotes in this string, you should revert to double quotes

Reply:
Double quotes have been added around Smith, John in all language files.